src/null-safety/understanding-null-safety.md

```diff
@@ -647,7 +647,8 @@ crashing.
 
 But if the value isn't `null`, it would be good to be able to move it over to
 the non-nullable side so you can call methods on it. Flow analysis is one of the
-primary ways to do this for local variables and parameters. We've extended type
+primary ways to do this for local variables and parameters
+(and private, final fields, as of Dart 3.2). We've extended type
```

Comment:
Nit: I find "private, final fields" harder to scan than "private final fields". IMHO the comma isn't necessary, for the same reason that "delicious fried plantains" doesn't need a comma.